File: app/src/main/java/me/zheteng/android/videosaver/saver/provider/OriginalUrlProvider.java
package me.zheteng.android.videosaver.saver.provider;

import me.zheteng.android.videosaver.saver.UrlProvider;

/**
 * 快手地址
 */
public class OriginalUrlProvider implements UrlProvider {
    String mTargetUrl;

    public OriginalUrlProvider(String original) {
        mTargetUrl = original;
    }

    @Override
    public String getTargetUrl() {
        return mTargetUrl;
    }
}

File: app/src/main/kotlin/me/zheteng/android/videosaver/parse/provider/OriginalUrlProvider.kt
package me.zheteng.android.videosaver.parse.provider

import me.zheteng.android.videosaver.parse.UrlProvider

/**
 * 只是存储原始地址
 */
class OriginalUrlProvider(private val mTargetUrl: String) : UrlProvider {

    override fun getTargetUrl(): String {
        return mTargetUrl
    }
}
